packages/containers/src/List/List.test.js

```diff
@@ -211,6 +206,21 @@ describe('Container List', () => {
 		// then
 		expect(dispatchActionCreator).toBeCalledWith('pagination:change', event, data, context);
 	});
+
+	it('should set the proper rowHeight', () => {
+		const rowHeight = {
+			table: 3,
+			large: 2,
+			tile: 1,
+		};
+		const wrapper = shallow(
+			<Container {...cloneDeep(settings)} items={items} rowHeight={rowHeight} />,
+			{ lifecycleExperimental: true },
+		);
+		const props = wrapper.props();
+		expect(props.displayMode).toBe('table');
+		expect(props).toMatchSnapshot();
```

Comment:
It's a been over-engineered, you just want to check the `rowHeight`, just check it instead of the whole props.